com.smallstep.Smallstep.yml

```diff
@@ -0,0 +1,93 @@
+id: com.smallstep.Smallstep
+runtime: org.freedesktop.Platform
```

Comment:
Since we have to compile `webkitgtk` anyway, there's no real reason to use the Gnome runtime here, and we can slim things down a bit by using the base runtime.

Reply:
I think we can just use the GNOME SDK from the start since we know we are going to move to it once wails supports webkit2gtk 4.1 which ships in in that by default IIRC.